### PR TITLE
Add Alpine Linux into Installation in Readme

### PR DESCRIPTION
**Pull request by jirutka**

### CODE REVIEW

**Comment by ariasuni** [comment]
Shouldn’t this be mentioning to enable edge/community first? (I’m not sure how Alpine repositories are named)

**Comment by jirutka** [comment]
exa is in stable community repository since Alpine v3.11 and Alpine’s community repository is typically enabled by default.

**Comment by ariasuni** [comment]
I downloaded Alpine Linux Extended x86_64 and installed it in a virtual machine, and I only had the main repository enabled.

I see now that exa is available on stable though, I was confused by the fact that the package search on their website is on edge by default.

File: README.md [hunk 1]
```diff
@@ -103,6 +103,12 @@ Some of the options accept parameters:
 exa is available for macOS and Linux.
 More information on how to install exa is available on [the Installation page](https://the.exa.website/install).
 
+### Alpine Linux
+
+On Alpine Linux, install the [`exa`](https://pkgs.alpinelinux.org/package/edge/community/x86_64/exa) package.
```
Comment (ariasuni) [inline README.md:108]: I think it’s better to mention that it’s in the community repository, and just link to Alpine’s wiki:

```suggestion
On Alpine Linux, [enable community repository](https://wiki.alpinelinux.org/wiki/Enable_Community_Repository) and install the [`exa`](https://pkgs.alpinelinux.org/package/edge/community/x86_64/exa) package.
```

**Pull request merged**